servembtiles.py

```diff
@@ -120,14 +122,15 @@ def __call__(self, environ, start_response):
             elif uri_field_count >= 3:  # expect:  zoom, x & y
                 try:
                     zoom = int(base_uri)
-                    if all((self.minzoom, self.maxzoom)) and not (self.minzoom <= zoom <= self.maxzoom):
+                    if None not in (self.minzoom, self.maxzoom) and not (self.minzoom <= zoom <= self.maxzoom):
```

Comment:
NOTE: This also resolve the bug where `minzoom` was `0` causing this statement to be false.

Reply:
Ok!  Yes, I see 0 would be a problem here, thanks!

(Sorry haven't looked at this in awhile...)